client/src/partials/templates/bhDateEditor.tmpl.html

```diff
@@ -0,0 +1,37 @@
+<!-- Date editor component view implementation-->
+<div 
+	class="form-group"	
+	ng-class="{'has-error' : $ctrl.validationTrigger && $ctrl.form.dateEditorValue.$invalid}" >
+
+	<!-- Input component implementation-->
+	<div class="input-group">
+	    <input
+	    	data-date-editor-input
+	    	name="dateEditorValue"
+	    	type="text"
+	    	class="form-control"	    	
+	    	uib-datepicker-popup="{{$ctrl.dateFormat}}"
+	    	ng-model="$ctrl.dateValue"
+	    	is-open="$ctrl.editMode"
+	    	min-date="$ctrl.minDate"
+	    	max-date="$ctrl.maxDate"
+	    	close-text="Close"
+	    	ng-disabled="!$ctrl.editMode"
+	    	show-button-bar="true"
+	    	required
+	    	/>
+	    <span class="input-group-btn">
+	    	<button id="set-date-btn" class="btn btn-default" type="button" ng-click="$ctrl.changeMode()" ng-disabled="$ctrl.editMode">
+	    		{{"DATEFINDER.SET" | translate}}
+	    	</button>
+	    </span>
+    </div> 
+
+    <!-- Error message implementation-->
+    <p class="help-block" ng-messages="$ctrl.form.dateEditorValue.$error" ng-show="$ctrl.validationTrigger">
```

Comment:
Why not use `ng-messages-include` to reduce the amount of translations/code we need to maintain?  Like this:
```html
<div ng-messages-include="partials/templates/messages.tmpl.html"></div>
```